recipes/minizip/all/conanfile.py

```diff
@@ -0,0 +1,138 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class MinizipConan(ConanFile):
+    name = "minizip"
+    description = "minizip is a zip manipulation library written in C "
+    "that is supported on Windows, macOS, and Linux."
+    topics = ("conan", "minizip", "compression")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/nmoinvaz/minizip"
+    license = "Zlib"
+    exports_sources = ["CMakeLists.txt", "patches/*"]
+    generators = "cmake_find_package"
+
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "compat": [True, False],
+        "zlib": [True, False],
+        "bzip2": [True, False],
+        "zstd": [True, False],
+        "lzma": [True, False],
+        "pkcrypt": [True, False],
+        "wzaes": [True, False],
+        "libcomp": [True, False],
+        "openssl": [True, False],
+        "libbsd": [True, False],
+        "brg": [True, False],
+        "signing": [True, False],
+        "compress_only": [True, False],
+        "decompress_only": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "compat": True,
+        "zlib": True,
+        "bzip2": True,
+        "lzma": True,
+        "zstd": True,
```

Comment:
```suggestion
        "with_zlib": True,
        "with_bzip2": True,
        "with_lzma": True,
        "with_zstd": True,
```

Reply:
I thinking if with libbsd should be prefixed to, because it is also a dependency that can be pulled by cmake [https://github.com/nmoinvaz/minizip/blob/3ecff07e3cf76b848b9d6b219302c1c5311aa281/CMakeLists.txt#L333](url).

Reply:
On on side, there already exists a minizip recipe with plain `zlib`, `bzip`...
So this might be breaking when people update their minizip version.
On the other side, that is always thing with updating: things change.
Changing to `with_xxx` would make the recipe more in line with the others.

Reply:
Pulled is probably not a good description, as cmake does not download libbsd but tries to find it.